packages/myst-frontmatter/src/frontmatter/validators.ts

```diff
@@ -533,7 +539,7 @@ export function validateAuthor(input: any, stash: ReferenceStash, opts: Validati
     output.nameParsed = validateName(value.name, incrementOptions('name', opts));
     output.name = output.nameParsed?.literal;
   } else {
-    validationWarning('author should include name', opts);
+    validationWarning('contributor should include name', opts);
```

Comment:
I wonder if this is less meaningful for the author now. For example if there is an error in frontmatter in the `authors` field, will the message here be talking about contributors? That could be misleading

Reply:
can a user now list `authors` and `contributors` in their frontmatter?